### PR TITLE
doc: 17.2.5 Release Notes

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/releases/quincy.rst [hunk 1]
```diff
@@ -5,6 +5,44 @@ Quincy
...
+==============
+This is the fifth hotfix release in the Quincy series. We recommend
+that all users update to this release.
+
+Notable Changes
+---------------
+
+* Treating daemons as `<daemon_type>.x` caused a crash
+in the Telemetry module since the current method does not cover a case
+where a daemon id is more complex, i.e. `<daemon_type>.x.y`.
```
Comment (anthonyeleven) [inline doc/releases/quincy.rst:18]: ID

File: doc/releases/quincy.rst [hunk 2]
```diff
@@ -5,6 +5,44 @@ Quincy
...
+When we parse the daemon type and daemon id, we should
+split it into a maximum of two pieces rather than splitting
+it by every `.` character. Specifying `1` in the Python
```
Comment (anthonyeleven) [inline doc/releases/quincy.rst:21]: s/it//
s/in/to/

**Review by anthonyeleven — APPROVED** [review]
suggestions left, but approved so I don't block

**Comment by jdurgin** [comment]
jenkins test docs

File: doc/releases/releases.yml [hunk 3]
```diff
@@ -15,6 +15,8 @@ releases:
   quincy:
     target_eol: 2024-06-01
     releases:
+      - version: 17.2.5
+        released: 2022-10-17
```
Comment (jdurgin) [inline doc/releases/releases.yml:19]: date is old

File: doc/releases/quincy.rst [hunk 4]
```diff
@@ -5,6 +5,44 @@ Quincy
...
+.split() function will limit the split to a maximum of two items.
+Fixes: https://tracker.ceph.com/issues/57700
```
Comment (jdurgin) [inline doc/releases/quincy.rst:23]: the fixes and signed-off-by lines aren't needed

File: doc/releases/quincy.rst [hunk 5]
```diff
@@ -5,6 +5,44 @@ Quincy
...
+Signed-off-by: Laura Flores <lflores@redhat.com>
+
+*  bea9f4b introduced a regression that makes the activate process
+take a very long time to complete.
+`_get_bluestore_info()` which calls `ceph-bluestore-tool` binary via
+subprocess is called in an exponential way while this is not needed.
+Fixes: https://tracker.ceph.com/issues/57627
+Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>
+
+* Resolves: https://tracker.ceph.com/issues/55605
```
Comment (jdurgin) [inline doc/releases/quincy.rst:33]: don't need to include this here, the short description in the changelog is good enough

File: doc/releases/releases.yml (hunk 3 above)
Comment (ljflores) [reply doc/releases/releases.yml]: Fixed

**Review by ljflores — APPROVED** [review]

File: doc/releases/quincy.rst [hunk 6]
```diff
@@ -5,6 +5,42 @@ Quincy
...
+  
+  Related tracker: https://tracker.ceph.com/issues/57700
+
+* A ceph-volume regression introduced in bea9f4b that makes the
+  activate process take a very long time to complete has been
+  fixed.
+  
+  Related tracker: https://tracker.ceph.com/issues/57627
+
+* An exception that occurs with with some NFS commands
```
[the remark this answers is not in the document]
Comment (ljflores) [reply doc/releases/quincy.rst]: Fixed

File: doc/releases/index.rst [hunk 7]
```diff
@@ -270,4 +271,4 @@ Release timeline
 .. _0.67.1: dumpling#v0-67-1-dumpling
 .. _0.67: dumpling#v0-67-dumpling
 .. _Dumpling:  dumpling
-   
+
```
Comment (neha-ojha) [inline doc/releases/index.rst:274]: intentional?

File: doc/releases/quincy.rst [hunk 8]
```diff
@@ -5,6 +5,42 @@ Quincy
 Quincy is the 17th stable release of Ceph.  It is named after Squidward
 Quincy Tentacles from Spongebob Squarepants.
 
+v17.2.5 Quincy
+==============
```
Comment (neha-ojha) [inline doc/releases/quincy.rst:9]: add a line after this

Comment (ljflores) [reply doc/releases/quincy.rst:9]: Fixed

**Review by jdurgin — APPROVED** [review]

**Comment by ljflores** [comment]
Doc rendered here: https://ceph--48532.org.readthedocs.build/en/48532/releases/quincy/

**Pull request merged**